automatic/awscli/legal/VERIFICATION.txt

```diff
@@ -0,0 +1,18 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+1.  Download the installer:
+
+    x32: https://nodejs.org/dist/v10.15.3/node-v10.15.3-x86.msi
+    x64: https://nodejs.org/dist/v10.15.3/node-v10.15.3-x64.msi
```

Comment:
These URLs do not seem to be correct. It is not node that this package will install.